### PR TITLE
🔀 feat/jackal to main

### LINKED ISSUE #58: Edit icons to the same color

### PR DESCRIPTION
**Pull request by cuprum-acid**
Merge branch with updated pictures (Changed "jackal" png low quality pictures and icons with SVG one)
Closes #58

### CODE REVIEW

File: src/lib/components/Footer.svelte
Comment (dprostiruk) [inline src/lib/components/Footer.svelte:1]: Nice😁 SVG components are necessary for sure😊

File: src/routes/+page.svelte
Comment (dprostiruk) [inline src/routes/+page.svelte:1]: Ok😁

File: src/lib/images/home_img/tech_logos.svg
Comment (dprostiruk) [inline src/lib/images/home_img/tech_logos.svg:1]: I can't find webstorm logo here😢

File: src/lib/images/footer_img/email.png
Comment (dprostiruk) [inline src/lib/images/footer_img/email.png:1]: Bro what is this😁

File: src/lib/images/about_img/feat_4.svg
Comment (dprostiruk) [inline src/lib/images/about_img/feat_4.svg:1]: It's alright🙂

File: src/lib/images/footer_img/email.svg
Comment (dprostiruk) [inline src/lib/images/footer_img/email.svg:1]: Nice email logo📧

File: src/lib/images/about_img/feat_3.svg
Comment (dprostiruk) [inline src/lib/images/about_img/feat_3.svg:1]: Calculator to count money🤑

File: src/lib/images/about_img/feat_2.svg
Comment (dprostiruk) [inline src/lib/images/about_img/feat_2.svg:1]: Not bad😁

File: src/lib/images/about_img/feat_1.svg
Comment (dprostiruk) [inline src/lib/images/about_img/feat_1.svg:1]: Good😍

File: src/lib/components/TechnologicalStack.svelte
Comment (dprostiruk) [inline src/lib/components/TechnologicalStack.svelte:1]: SVG again😁

**Review by dprostiruk — APPROVED** [review]
Good job😁 SVG components are awesome👍

File: src/routes/+page.svelte
Comment (axonstan) [reply src/routes/+page.svelte:1]: Awesome comment😁

**Review by axonstan — APPROVED** [review]

**Pull request merged**